rfcs/trust-and-safety/20230109-trust_and_safety_preliminary_overview_and_exploration.md

```diff
@@ -0,0 +1,285 @@
+knowledge) that have user generated content automatically scan that content for
+illegal materials. The other providers are either unlikely to ever include
+illegal materials (GLAM institutions) or manually review each submission by hand
+(WordPress Photo Directory). This may not always be the case. One of the future
+goals for Openverse is for websites (and WordPress sites, in particular) to be
+able to automatically add themselves as providers of openly licensed content.
+Once that is possible, then it will become even more imperative that Openverse
+scans materials for illegal materials. Even before then, however, I think it is
+prudent for us to consider doing it earlier because running the risk of
+distributing illegal content (which is also universally heinous content) is not
+something Openverse ever wants to be involved in, even if it is only because one
+of our upstream content providers made a mistake.
+
+### Openverse as a content "host"
+
+While Openverse does not, to a meaningful extent, "host" the original content
+ingested from providers, it does have specially cached resources that, if not
+appropriately managed, could persist, even if a result is removed. For example,
+our thumbnails are heavily cached on infrastructure that we control (or, more
+accurately, infrastructure that exists under a PaaS account we're responsible
+for). Removing a result from showing in search is not sufficient for completely
+removing it from Openverse's distribution of the content. A particularly
+difficult corner case for this is if a result is removed from the upstream
+provider for content moderation reasons and then removed from our catalogue upon
+re-ingestion, the cached thumbnail persists in the current implementation.
+Maintaining the thumbnail cache is a critical aspect of ensuring that Openverse
+does not continue to accidentally be a distributor of content we do not wish
+to/have a legal obligation not to. Several of the technical requirements
+mentioned below involve juggling various important, interconnected, and
+interrelated caches: caches of image classification data obtained at search
+time, link status caches, and thumbnail caches.
+
+### GLAM institutions and the relationship of historical collections to sensitive visual and textual materials
+
+Museum and library collections often include historical material with sensitive
+content. For example, a museum may hold, catalogue, and distribute a photograph
+with its original caption even if the caption describes the subject in an
+offensive or inaccurate way. Especially common examples of this are images of
+racialised people with original captions that use slurs or other derogatory
+language to describe the subject. I (Sara) do not know if this is something that
+currently exists in Openverse, but it is something we could discuss with
+providers if we discovered it to ensure that we are capturing the relevant
+metadata. For example, some providers may include a note clarifying that a
```

Comment:
In general, I agree that we should show the provider's notes.
The implementation details can be complicated by the way the notes are displayed on the providers' sites or in their APIs and the way we ingest that data. I tried searching for some racial slurs on Openverse. One example I found is from Boston Public Library. We have ingested this item from their Flickr stream, which does not have any notices. However, the same item is also hosted on digitalcommonwealth.org, and their it has a notice banner at the top of the page: https://www.digitalcommonwealth.org/search/commonwealth:fq977w05r. I'm not sure if it's returned from the API, though.

Reply:
That's a great example of the kind of thing we should do. For sensitive textual content we could pretty reliably attach a notice of our own for items like that.

That specific one does not look unique to the item though, it seems like a generic one (like we could do just by scanning textual content for specific words). If there are GLAM providers that do have handwritten catalogue notes that are accessible to us in this vein we should include them though.